handbook/communication/rfcs/index.md

```diff
@@ -1,139 +1,173 @@
 
-- Google Docs does not support markdown and does not provide an ideal experience for embedding code snippets.
-  - Many RFCs don't require code snippets because they are higher level than "how are we going to implement this in code".
-  - Even RFCs about technical implementation details don't seem to be hindered by Google Docs lack of good snippet support.
-    - RFC 16 discusses the structure of the search API without getting into the weeds of the actual API.
-    - RFC 29 also discusses the structure of the search API without explicit code snippets. It is actually a rewrite of a Notion doc that did have a lot of snippets, and the Google Doc is actually much easier to understand.
-  - You can still put code snippets in Google Docs and get your point across. Using a monospace font helps.
-  - You can link to another tool where the code snippet can be better formatted (e.g. Gist, PR).
+We have a set of status labels we’ve found helpful in the past:
 
-Rejected alternatives:
+- **WIP**: The author is still drafting the RFC and it’s not ready for review.
+- **Review**: The Review label is used when the RFC is ready for comments and feedback.
+- **Approved**: When the RFC is for the purpose of making a decision, the Approved label indicates that the decision has been made.
+- **Implemented**: When the RFC is for the purpose of making a decision, the Implemented label indicates that the RFC’s proposal has been implemented.
+- **Closed**: When the RFC is for the purpose of collaboration or discussion but not necessarily to make a decision or propose a specific outcome that will eventually become Implemented, the Closed label indicates that the RFC is no longer an active collaborative artifact.
+- **Abandoned**: When the RFC is for the purpose of making a decision, and there are no plans to move forward with the RFC’s proposal, the Abandoned label indicates that the RFC has been purposefully set aside.
 
-- [Notion](https://www.notion.so)
-  - Comments on documents are too hidden and easy to miss.
-  - We would have to manage a separate account system or pay for SSO.
-- Markdown in git
-  - We actually [tried this for over a year](https://github.com/sourcegraph/docs-private) and stopped.
-  - GitHub PRs don't provide a good commenting experience.
-    - Can only comment on lines, not words.
-    - Suggestions are cumbersome to make/read.
-    - Even with recent improvements to collapse old threads, the PR activity page becomes cumbersome after a lot of discussion has happened on a document.
-  - It is higher friction to author and/or contribute to multiple RFCs due to either (1) the necessary git workflow or (2) the clunky editing experience in GitHub's UI.
+While these labels suggest that RFCs move through series of prescribed stages, this isn’t always the case. RFCs are low-process and can be used for all kinds of purposes that may not have an explicit, actionable outcome.
```

Comment:
maybe add a sentence here about how the RFC should make it clear (beyond just the label) what the status is and expected process is